Rubberduck.Deployment/InnoSetup/Rubberduck.Installer.Build.iss

```diff
@@ -522,6 +537,14 @@ begin
     result := true;
 end;
 
+function CheckUseLegacyWorkloadConfig():boolean;
+begin
+  if WorkloadOptionPage.Values[0] then
```

Comment:
I am not sure if we should be referencing the option page directly here. In other similar functions, you will note those use variables, without any references to the option page. I mention that because I am unsure whether it's reliable to keep referencing a page from the non-code section as it may not be in scope when that portion runs even though it's a global variables. I would change it to use `UseLegacyWorkloadConfig` variable, and use the next button click event to assign the value to that variable.